types/office-js-preview/index.d.ts

```diff
@@ -92356,6 +92356,8 @@ declare namespace Word {
      *
      * @remarks
      * [Api set: WordApi 1.4]
+     *
+     * Important: To learn more about which fields can be inserted, see the Word.Range.insertField API introduced in requirement set 1.5. Support for managing fields is similar to what's available in the Word UI. The Word UI on the web primarily only supports existing fields as read-only. To learn more, see {@link https://support.microsoft.com/office/d8f46094-13c3-4966-98c3-259748f3caf1 | Field codes in Word for the web}. To learn more about Word UI clients that more fully support fields, see the product list at the beginning of {@link https://support.microsoft.com/office/c429bbb0-8669-48a7-bd24-bab6ba6b06bb | Insert, edit, and view fields in Word}.
```

Comment:
"The Word UI on the web primarily only supports existing fields as read-only." For this statement, it works for most fields. But at least for existing Word.FieldType.addin fields, they're supported to be editable. This is done by Word Online team along with the release of WordApi 1.5.

Reply:
@xiruatms Thanks, I'll incorporate this information.